Riot/Modules/Spaces/SpaceMenu/SpaceMenuListItemViewData.swift

```diff
@@ -16,6 +16,14 @@
 
 import Foundation
 
+/// Possible action ID related to a `SpaceMenuListViewCell` view data
+enum SpaceMenuListItemActionId {
```

Comment:
Yeah, this change to use the enum looks great to me! Small thought: the `Id` suffix is possibly irrelevant now there's no string behind it. Looking at `didSelectItemWithId: actionId` where it could all just become `didSelectItemWithAction: action` for example.